main.cpp

```diff
@@ -52,106 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: slider
+//  action 1: the slider goes up and down
+slider.moveUpAndDown();
+//  action 2: the slider reutrn to previous position
+slider.undoPreviousPosition();
+//  action 3: the slider return to default position
+slider.defaultPosition();
+  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: button
+//  action 1: the button extends
+button.extend();
+//  action 2: the button is glow
+button.glow();
+//  action 3: the button is changing his text
+button.changingText();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: speaker
+//  action 1: the speaker make sounds
+speaker.makeSounds();
+//  action 2: the speaker light change color
+speaker.changeLightColor();
+//  action 3: the speaker shut down
+speaker.shutDown();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: alarmClock
+//  action 1: the clock is ringing
+alarmClock.ring();
+//  action 2: the clock is repeat every 5 min
+alarmClock.repeat();
+//  action 3: the clock is stop ringing
+alarmClock.stop();
+
+//  5) 
+//  Noun: fan
+//  action 1: the fan is spinning 
+fan.spin();
+//  action 2: the fan blows cold air
+fan.blowColdAir();
+//  action 3: the fan blows hot air
+fan.blowHotAir();
+  
+//  6) 
+//  Noun: computer
+//  action 1: computer get into sleep mode
+computer.sleepMode();

```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise